test-expiration

```diff
@@ -5,11 +5,25 @@ set -euxo pipefail
 TEST_CONFIG=${TEST_CONFIG:-./test-config.sh}
 . "$TEST_CONFIG"
 
+# The --show-keys option is only available on recent GPG2. Use
+# --list-keys with --keyring and various options otherwise.
```

Comment:
This comment is no longer accurate.

Reply:
Ah, I misread the comment. It's accurate but maybe:

```suggestion
# --list-keys with --keyring and various options instead.
```

Reply:
Good call. I glossed over that comment and thought it was basically right, but this is better.